### PR TITLE
DOCS-16422 clarifying write blocking behavior

### PR DESCRIPTION
**Pull request by ltran-mdb2**
## DESCRIPTION

Clarifying when writes are blocked with mongosync 

## STAGING

https://preview-mongodbltranmdb2.gatsbyjs.io/cluster-sync/DOCS-16422/reference/mongosync/#write-blocking

## JIRA

https://jira.mongodb.org/browse/DOCS-16422

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=654becfe5b686d6bd97d3f7f

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Is this free of spelling errors?
- [x] Is this free of grammatical errors?
- [x] Is this free of staging / rendering issues?
- [x] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

**Review by ianf-mongodb — COMMENTED** [review]
Hi @ltran-mdb2, thanks for picking this up, I left a couple suggestions for consideration.

File: source/reference/mongosync.txt [hunk 1]
```diff
@@ -139,9 +139,11 @@ To stop syncing, use the :ref:`commit <c2c-api-commit>` command on the
 destination cluster to convert indexes and finalize the changes. While
 ``commit`` is running:
 
-- Writes are blocked on the source cluster.
-- Writes are blocked on the destination cluster until ``mongosync``
-  begins index validation.
+- If you enable write-blocking, ``commit`` blocks writes on the source
```
[the remark this answers is not in the document]
Comment (ltran-mdb2) [reply source/reference/mongosync.txt]: I think it makes sense to put this information under the write blocking section and changed accordingly

**Comment by ltran-mdb2** [comment]
Thanks for the feedback @ianf-mongodb ! Could you take another look when you get a chance?

File: source/includes/fact-write-blocking-enable.rst [hunk 2]
```diff
@@ -1,7 +1,9 @@
 When write-blocking is enabled, ``mongosync`` blocks writes:
 
-- On the destination cluster during sync
-- On the source cluster while committing
+- On the destination cluster during sync.
+- On the source cluster while ``commit`` is running.
+- On the destination cluster while ``commit`` is running, until
```
Comment (ianf-mongodb) [inline source/includes/fact-write-blocking-enable.rst:5]: To be clear what I meant was move the entire commit section below the write block section. This way the concept of write blocking is introduced first and then the commit content is introduced. 

I think combining them is slightly confusing, as these bullets are contradicting:

- On the destination cluster during sync.
- On the destination cluster while ``commit`` is running, until ``mongosync`` begins index validation.

File: source/includes/fact-write-blocking-enable.rst [hunk 3]
```diff
@@ -1,7 +1,9 @@
 When write-blocking is enabled, ``mongosync`` blocks writes:
 
-- On the destination cluster during sync
-- On the source cluster while committing
+- On the destination cluster during sync.
```
Comment (ianf-mongodb) [inline source/includes/fact-write-blocking-enable.rst:3]: I would also move the sentence `Write-blocking is not enabled by default.` to the top of this include and remove it from[ line 226 ](https://github.com/mongodb/docs-cluster-to-cluster-sync/pull/182/files#diff-a61a44aa44c4308e9fcfa894effe294e608468bf2a1e3cf752da6b039ef5f2beR226). This is a pivotal piece of information that the user must understand, even when not using reverse sync.

**Review by ianf-mongodb — COMMENTED** [review]
Hi @ltran-mdb2, thanks for the updates I left a couple clarifying comments.

File: source/includes/fact-write-blocking-enable.rst (hunk 2 above)
Comment (ltran-mdb2) [reply source/includes/fact-write-blocking-enable.rst]: ah my bad -  I moved the commit section below write-blocking and uncombined the two!

File: source/includes/fact-write-blocking-enable.rst (hunk 3 above)
Comment (ltran-mdb2) [reply source/includes/fact-write-blocking-enable.rst:4]: Done!

**Comment by ltran-mdb2** [comment]
Thanks for your suggestions @ianf-mongodb! Could you take another look when you get a chance?

File: source/includes/fact-write-blocking-enable.rst [hunk 4]
```diff
@@ -1,7 +1,8 @@
-When write-blocking is enabled, ``mongosync`` blocks writes:
+MongoDB does not enable write-blocking by default. If you enable
```
Comment (ianf-mongodb) [inline source/includes/fact-write-blocking-enable.rst:1]:
```suggestion
``mongosync`` does not enable write-blocking by default. If you enable
```

**Review by ianf-mongodb — APPROVED** [review]
LGTM % 1 suggestion thanks @ltran-mdb2!

**Comment by ltran-mdb2** [comment]
LGTM from Alex Komyagin over slack

**Pull request merged**